src/Models/Types/Generated/3DScenesConfiguration-v1.0.0.d.ts

```diff
@@ -273,6 +275,7 @@ export interface IADXTimeSeriesConnection {
 export interface IDataHistoryBasicTimeSeries {
     id: string;
     expression: string;
+    propertyType?: PropertyValueType;
```

Comment:
did you manually add this? It's odd that the type here is not matching to the schema you defined. 
Did you use this doc to update the schema?
https://github.com/microsoft/iot-cardboard-js/wiki/Coding-guidelines~Schema-management

Reply:
ha yea I did - will generate.